src/index.ts

```diff
@@ -156,6 +165,10 @@ interface ISQLite {
   ) => StatementResult;
   detach: (mainDbName: string, alias: string) => StatementResult;
   transaction: (dbName: string, fn: (tx: Transaction) => boolean) => void;
+  asyncTransaction: (
```

Comment:
Could you add the asyncTransaction method to IDBConnection to keep it feature equivalent with ISQLite?